__tests__/login.test.js

```diff
@@ -0,0 +1,36 @@
+// LoginPage.test.js
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import LoginPage from '../pages/auth/login/index';
+import { LayoutProvider } from '../layout/context/layoutcontext';
+import { UserContext } from '../src/context/usercontext';
+import { useRouter } from 'next/router';
+import { publicRuntimeConfig } from '../next.config';
+
+jest.mock('next/router', () => ({
+  useRouter: jest.fn(),
+}));
+
+jest.mock('next/config', () => () => ({
+  publicRuntimeConfig: {
+    contextPath: '',
+  },
+}));
+
+describe('LoginPage', () => {
+  it('renders without crashing', () => {
+    useRouter.mockImplementation(() => ({
+      push: jest.fn(),
+    }));
+
+    render(
+      <LayoutProvider>
+        <UserContext.Provider value={{ user: null, setUser: jest.fn() }}>
+          <LoginPage />
+        </UserContext.Provider>
+      </LayoutProvider>
+    );
+    
+    expect(screen.getByText('Welcome, Isabel!')).toBeInTheDocument();
```

Comment:
I noticed the "Welcome, Isabel!" when I ran the server locally. Can you explain what your intention is with this? From my understanding, the user is not logged in when they are on the login page, so we wouldn't know their name

Reply:
This is an artifact from the template Jeff used to pull something in quickly. As Tristan and Katie are going to rework it I figured I'd leave it to be a sore thumb.

Reply:
Gotcha 👍